test/cmd/create.sh

```diff
@@ -75,8 +75,19 @@ os::cmd::expect_success "_output/oshinko delete sally"
 # metrics
 os::cmd::expect_success "_output/oshinko create klondike --metrics=true"
 os::cmd::try_until_success "oc get service klondike-metrics"
+os::cmd::try_until_text "oc log dc/klondike-m" "with.*metrics"
```

Comment:
once the openshift-spark PR is merged, the wildcard here and below can be replaced with a specific string